dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -132,41 +132,81 @@ public void createTest() throws Exception {
     }
 
     @Test
-    public void createWithParentTest() throws Exception {
+    public void createSubCommunityUnAuthorizedTest() throws Exception {
         //We turn off the authorization system in order to create the structure as defined below
         context.turnOffAuthorisationSystem();
 
         // Create a parent community to POST a new sub-community to
         parentCommunity = CommunityBuilder.createCommunity(context)
                                           .withName("Parent Community")
                                           .build();
+
         context.restoreAuthSystemState();
 
         ObjectMapper mapper = new ObjectMapper();
         CommunityRest comm = new CommunityRest();
         // We send a name but the created community should set this to the title
         comm.setName("Test Sub-Level Community");
 
+        // Anonymous user tries to create a community.
+        // Should fail because user is not authenticated. Error 401.
+        getClient().perform(post("/api/core/communities")
+            .content(mapper.writeValueAsBytes(comm))
+            .param("parent", parentCommunity.getID().toString())
+            .contentType(contentType))
+                   .andExpect(status().isUnauthorized());
+
+        // Non-admin Eperson tries to create a community.
+        // Should fail because user doesn't have permissions. Error 403.
+        String authToken = getAuthToken(eperson.getEmail(), password);
+        getClient(authToken).perform(post("/api/core/communities")
+            .content(mapper.writeValueAsBytes(comm))
+            .param("parent", parentCommunity.getID().toString())
+            .contentType(contentType))
+                            .andExpect(status().isForbidden());
+    }
+
+    @Test
+    public void createSubCommunityAuthorizedTest() throws Exception {
```

Comment:
👍 This is a good test to see, as it shows we can indeed do deeper authorization checks than "user is admin" with the annotations.